packages/nova-react-test-utils/package.json

```diff
@@ -11,35 +11,49 @@
     "just": "monorepo-scripts"
   },
   "peerDependencies": {
+    "@storybook/addon-actions": "^7.6",
+    "@storybook/preview-api": "^7.6",
     "@storybook/react": "^7.6",
-    "react": "^17.0.2 || ^18"
+    "react": "^17.0.2 || ^18",
+    "@apollo/client": "^3.4",
+    "@graphitation/apollo-mock-client": "^0.11.0",
+    "relay-compiler": "^12.0.0",
```

Comment:
This should probably be aligned with 16 || 17?

Reply:
Maybe that's not so easy because of duct-tape that needs 12?

So perhaps it's 12 || 16 || 17?

Reply:
I removed it in the end, it was added as peer dep for one of graphitation packages but with them being now in peerDeps of test utils, it is not needed to add them again here